sys/parsoid.js

```diff
@@ -77,7 +77,7 @@ function _dependenciesUpdate(hyper, req, newContent = true) {
     const rp = req.params;
     return mwUtil.getSiteInfo(hyper, req)
     .then((siteInfo) => {
-        const baseUri = siteInfo.baseUri.replace(/^https?:/, '');
+        const baseUri = siteInfo.baseUri.toString().replace(/^https?:/, '');
```

Comment:
I wonder if this should be fixed in `getSiteInfo` and stringified right away, given that it's the base URI. What do you think, @Pchelolo ?

Reply:
Ye, let's do it in the getSiteInfo. Agreed.

Reply:
Done.